.github/ISSUE_TEMPLATE/01_bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: '🐛 Bug Report'
+about: Please always create a discussion first 👇
+labels: bug
+---
+
+<!--
+  ⛔ STOP!
+
+  The Snowpack project reserves issues for confirmed/triaged bugs ONLY.
+  Please start by creating a discussion at https://github.com/pikapkg/snowpack/discussions/new?category_id=16417352
+
+  Discussion already exists? Please replace all <!-- placeholders --\> below
+-->
+
+**Follow up**
+
+This is a follow up to
+
+- <!-- add URLs to discussion(s) / issue(s) / pull request(s) -->
+
+**Versions**
+
+<!-- Paste the output of `npm ls | grep snowpack` or `yarn list --pattern snowpack`  -->
+
+**Environment**
+
+- [ ] npm: <!-- `npm --version` -->
+- [ ] yarn: <!-- `yarn --version` -->
+- [ ] operation system:

```

Comment:
something that would be great: a `npx snowpack debug-info` (or similar) command, which would output everything we need. We could ask folks do just paste the output of that here.

For the time being, anything else we should ask about?